source/supply_api.html.md

```diff
@@ -410,6 +410,8 @@ customer.phone\*\* | string | Phone number of the customer if we have it | N
 business | object | JSON business object | Y
 business.businessID | string | ID of the business (pro) | Y
 business.name | string | Business name | Y
+price | string | Price estimate for the job | N
+leadType | string | Type of lead that Thumbtack is sending | N
```

Comment:
It would be good to document what are the possible values this field could have

Reply:
sure. Great idea.